renovate-config.json

```
@@ -48,6 +48,11 @@
         "https://github.com/ckeditor/ckeditor5/"
       ],
       "groupName": "ckeditor5"
+    },
+    {
+      "matchPackageNames": ["xmlsec"],
+      "allowedVersions": "!/1\\.3\\.14/",
```

This will look for a literal backslash, followed by any character, followed by 3, etc.
```suggestion
      "allowedVersions": "!/1\.3\.14/",
```
With a single backslash, it is escaping the `.` to look for a literal period character.